deliverable1/vehicleConfigs/Jaguar2015XE (Apollo 5.0, many sensors).json

```diff
@@ -0,0 +1,180 @@
+[
```

Comment:
People who try to run the test should add those vehicles into their simulation. Otherwise, they will face a hanging issue.